diff --git a/pq-crypto/sike_r2/config.h b/pq-crypto/sike_r2/config.h
index 6199e5a7..a5e9104c 100644
--- a/pq-crypto/sike_r2/config.h
+++ b/pq-crypto/sike_r2/config.h
@@ -65,6 +65,15 @@ typedef uint16_t hdigit_t; // Unsigned 16-bit digit
 #define LOG2RADIX 6
 typedef uint64_t digit_t;  // Unsigned 64-bit digit
 typedef uint32_t hdigit_t; // Unsigned 32-bit digit
+#elif defined(__powerpc64__)
+// hmaarrfk: 2021/01/09
+// Ok, it isn't really ARM64, but I'm going to fake it
+// until upstream decides to support this correctly
+#define TARGET TARGET_ARM64
+#define RADIX 64
+#define LOG2RADIX 6
+typedef uint64_t digit_t;  // Unsigned 64-bit digit
+typedef uint32_t hdigit_t; // Unsigned 32-bit digit
 #else
 #error-- "Unsupported ARCHITECTURE"
 #endif
